asv/repo.py

```diff
@@ -152,6 +166,7 @@ class NoRepository(Repo):
     """
 
     dvcs = "none"
+    _default_branch = "master"
```

Comment:
NoRepository should be OK; it shouldn't need the branch value, since 
it's not checking out code or saving any results.